XCUITests/PhotonActionSheetTest.swift

```diff
@@ -63,6 +62,7 @@ class PhotonActionSheetTest: BaseTestCase {
     // Disable issue #5554, More button is not accessible
     /*
     // Test disabled due to new implementation Bug 1449708 - new share sheet
+    // User flow slightly different with the browser as a selectable instead of Send Tab
```

Comment:
I would say to remove the test if the user flow has changed. It does not make sense to have these disabled tests, would like to use this tasks to re-enable tests to also verify current behaviour and remove/clean old code. It is easier to update the test now than re-visiting it later, easy to forget :(

Reply:
Will do, will remove.